.github/workflows/integration-tests.yml

```diff
@@ -33,6 +35,8 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
+    - name: Install latest NPM
+      run: npm install -g npm@latest
```

Comment:
I think setting the version of npm in scripts like `benchmark-integration-test-direct.sh` or `checks-and-unit-tests.sh` would be problematic as since the docs mention developers to run those scripts before pushing a change, we might end up installing an unnecessary version of npm in the developer's machine when they run the script.

Reply:
The idea of the script would be to install a newer version only if required, otherwise not make any changes. Ideally I would like to centralise for the build the version of npm required to a single point rather than replicating it across the different sut testing

Reply:
I have added a script for checking prerequisites. However I haven't added it to npm's preinstall script as there is a bug in npm (https://github.com/npm/cli/issues/2660) due to which the preinstall script gets executed only after installing dependencies (whereas we want it to be executed before installing any dependency). I think the way to go is to update the contributor docs (https://hyperledger.github.io/caliper/v0.5.0/contributing/#bootstrapping-the-caliper-repository) to run the script before installing dependencies. I would like to know if there is any other way we could proceed.